mkdocs-project-dir/docs/howto.md

```diff
@@ -475,15 +490,14 @@ the nodes that exist in that cluster.
 
 ```
 [uccacxx@login12 ~]$ nodetypes
-Unknown node type: util02
-Unknown node type: util05
-    2 type * nodes: 36 cores, 188.4G RAM
+    3 type * nodes: 36 cores, 188.4G RAM
     7 type B nodes: 36 cores,   1.5T RAM
-   66 type D nodes: 36 cores, 188.4G RAM
+    1 type D nodes: 36 cores, 172.7G RAM
+   65 type D nodes: 36 cores, 188.4G RAM
```

Comment:
The D and H node changes weren't really necessary: temporarily a node of each is reporting a slightly different amount of memory, so splits into 1 and 65 instead of 66. I'd probably leave those out and only update the number of login nodes.